doc/src/manual/variables-and-scoping.md

```diff
@@ -99,6 +99,19 @@ julia> module E
 ERROR: cannot assign variables in other modules
 ```
 
+Each top-level expression has an invisible scope block around it. If a top-level expression contains a variable declaration with keyword `local`, then that variable is in the local scope of that expression. An example is to declare `local x` in a `begin` or `if` block at the top-level:
```

Comment:
> Each top-level expression has an invisible scope block around it.

I think that statement is a bit confusing, since toplevel expressions don't introduce an actual scope block like `let` or `while` where new assignments are always scoped by default. We should make clear that this only affects variables with an explicit `local` declaration.

Reply:
This is why I don't feel like I can clearly explain what's happening here. Is there a scope or not? If there isn't a scope, then what is the variable local to? Maybe there's a way to explain it, but if so, I don't know how to.

Reply:
I think I would mostly leave out the word scope here and just say that `local` inside top-level expressions means that the variable won't be accessible outside that expression.

Reply:
@simeonschaub Since the usage of scope seems to be confusing here I removed it.